underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -131,6 +131,12 @@ public void connectFromWorker(Configuration conf, String hostname) {
   public OutputStream create(String path) throws IOException {
     LOG.debug("Create method: {}", path);
     String newPath = path.substring(Constants.HEADER_SWIFT.length());
+    if (newPath.endsWith("_SUCCESS")) {
```

Comment:
This behavior is interesting. Could you explain it a bit more? When `path/_SUCCESS` file is created by Spark, does Spark also expects a placeholder for `path` indicating the directory? 
If so, when Spark directly runs on top of Swift, does it create both the objects for `path` and `path/_SUCCESS`? I wonder what's the difference between the file creation APIs between Alluxio and Swift.